TestApp/upgrade-appcenter-npm-packages.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-npm uninstall --save appcenter appcenter-analytics appcenter-crashes
+npm uninstall --save appcenter appcenter-analytics appcenter-crashes appcenter-link-scripts
```

Comment:
Why we should uninstall appcenter-link-scripts package?

Reply:
because it is used when app center is linked from local sources.